contrib/sideload_csv2json.py

```diff
@@ -0,0 +1,92 @@
+from copy import deepcopy
+import csv
+import json
+import ast
+
+startrow = 0
+endrow = 1000000000
+filename = "Starling-Caption-Credit.csv"
+config_filename = "Starling-Caption-Credit-config.json"
+
+# Load config
+with open(config_filename, "r") as config_file:
+    config = json.load(config_file)
+
+with open(filename, newline="\n", encoding="utf8") as csvfile:
```

Comment:
```suggestion
with open(filename, newline="", encoding="utf8") as csvfile:
```

`newline=""` is required according to the csv library docs: https://docs.python.org/3/library/csv.html#csv.reader